### PR TITLE
feat: add autofix for PLR0402

### PR DESCRIPTION
**Pull request by spaceone**
Issue #2199

### CODE REVIEW

File: src/rules/pylint/rules/use_from_import.rs [hunk 1]
```diff
@@ -31,11 +37,19 @@ pub fn use_from_import(checker: &mut Checker, alias: &Alias) {
...
             name: name.to_string(),
         },
         Range::from_located(alias),
-    ));
+    );
+    if checker.patch(diagnostic.kind.rule()) {
+        diagnostic.amend(Fix::replacement(
+            format!("from {module} import {asname}"),
+            import_stmt.location,
+            alias.end_location.unwrap(),
```
Comment (charliermarsh) [inline src/rules/pylint/rules/use_from_import.rs:51]: Should this not be `import_stmt.end_location.unwrap()`?

Comment (spaceone) [reply src/rules/pylint/rules/use_from_import.rs]: OK, changed. it should be the same.
I also fixed a special case when there are multiple imports in that import statement.

**Comment by spaceone** [comment]
@charliermarsh Off-Topic:
I just merged the branch which does our tab to spaces migration and many more ruff-enhanced things ([merge commit: 1,969 changed files with 203,016 additions and 203,292 deletions](https://github.com/univention/univention-corporate-server/commit/2e5688d5636a453b266ddf6a8315552ce6862383)).
If your are interested: this is out [pyproject.toml](https://github.com/univention/univention-corporate-server/blob/5.0-3/pyproject.toml).

Therefore I wanted to thank you for your effort in this project!!! The raw tabs-to-spaces migration was done with `autopep8` but `ruff` really helped me with the docstrings, which was not correctly done by `autopep8` and it made a lot of useful code refactoring things.

And I also wanted to appreciate your kindness in our collaboration: you took every issue of me serious and communicated clearly. You gave good hints in MRs. I enjoyed working with you so far!

I hope I will (have more time to) learn rust and contribute back to ruff in the future.
Best regards
spaceone

**Comment by charliermarsh** [comment]
@spaceone - That's amazing! Thank you so much for this kind note, and I'm really glad that Ruff is working well for you. I've enjoyed our collaboration too. You filed a lot of issues 🤣 and found tons of edge cases -- I appreciate _your_ contributions too (and I'm glad we could fix most of those issues!).

**Pull request merged**